OpenRA.Mods.Cnc/FileFormats/LZOCompression.cs

```diff
@@ -67,7 +67,7 @@
 
 using System;
 
-namespace OpenRA.Mods.Common.FileFormats
```

Comment:
https://de.wikipedia.org/wiki/Lempel-Ziv-Oberhumer is probably a generic algorithm.

Reply:
I guess so, but it's only used by ShpTD currently.
@pchote what do you think?

Reply:
If its not used by anything in `Mods.Common`, and theres no obvious need for it to be used by anything in `Mods.Common` in the future then it doesn't belong in `Mods.Common`.

Reply:
Uhm...

https://github.com/OpenRA/OpenRA/blob/4dd08d9dc2d4f8bf1a6407807211780e156befac/OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs#L295 is the sole usecase currently.

TD SHPs use LCW, not LZO.

Reply:
FWIW, LZO could've been added as a common package next to zip based on this.

Reply:
> TD SHPs use LCW, not LZO.

Yeah, guess i was too tired that day.

> FWIW, LZO could've been added as a common package next to zip based on this.

I'm not in the mood to get my head around that, let alone spend the time on it. And we can still move it back whenever anyone - whether us or some 3rd-party contributor - finds the time and motivation to tackle this.